src/Microsoft.AspNetCore.Sockets.Client/Connection.cs

```diff
@@ -28,58 +29,110 @@ private Connection(Uri url, ITransport transport, IChannelConnection<Message> tr
             _transportChannel = transportChannel;
         }
 
-        public ReadableChannel<Message> Input => _transportChannel.Input;
-        public WritableChannel<Message> Output => _transportChannel.Output;
-
-        public void Dispose()
+        private ReadableChannel<Message> Input => _transportChannel.Input;
+        private WritableChannel<Message> Output => _transportChannel.Output;
+        
```

Comment:
I really need to update the [Trailing Whitespace Flagger](https://marketplace.visualstudio.com/items?itemName=PawelKadluczka-MSFT.TrailingSpaceFlagger) to VS2017